new-account-trust-policy.py

```diff
@@ -151,15 +151,24 @@ def get_partition():
     return get_caller_identity()['Arn'].split(':')[1]
 
 
-def main(role_arn, role_name, trust_policy):
+def main(
+    role_arn,
+    role_name,
+    trust_policy,
+    botocore_cache_dir=BOTOCORE_CACHE_DIR,
```

Comment:
The script is setup to allow a user to run it from the command line. This parameter can be set by the user through an ENV, similar to the LOG_LEVEL, `BOTOCORE_CACHE_DIR = os.environ.get('BOTOCORE_CACHE_DIR')`.